app/views/moderations/_reactions.html.erb

```diff
@@ -1,13 +1,10 @@
 <h2>Add a reaction</h2>
 <button class="reaction-button <%= Reaction.cached_any_reactions_for?(moderatable, current_user, "like") ? "reacted" : "" %>" data-reactable-id="<%= moderatable.id %>" data-category="like" data-reactable-type="Article">
   <%= image_tag("favorite-heart-outline-button.svg", alt: "Thumbs down emoji") %>
-  <img class="reacted-emoji" src="<%= asset_path("emoji/emoji-one-heart.png") %>" alt="Thumbs down emoji" />
 </button>
 <button class="reaction-button <%= Reaction.cached_any_reactions_for?(moderatable, current_user, "thumbsdown") ? "reacted" : "" %>" data-reactable-id="<%= moderatable.id %>" data-category="thumbsdown" data-reactable-type="Article">
-  <%= image_tag("emoji/emoji-one-thumbs-down-gray.png", alt: "Thumbs down emoji") %>
-  <img class="reacted-emoji" src="<%= asset_path("emoji/emoji-one-thumbs-down.png") %>" alt="Thumbs down emoji" />
```

Comment:
Taking slightly different approach here to simplify things: instead of having two assets (one grayscale and one colored) for two states (default & clicked) I just run with one asset (colored) as we indicate its state in different ways with CSS anyway.